api/src/routers/review.js

```diff
@@ -0,0 +1,85 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const reviewRouter = express.Router();
+
+//Returns all reviews.
+reviewRouter.get("/", async (req, res, next) => {
+  try {
+    const allReviews = await knex("review");
+
+    if (allReviews.length === 0) {
+      return res.status(404).json({ message: "No reviews found" });
+    }
+
+    res.json(allReviews);
+  } catch (error) {
+    next(error);
+  }
+});
```

Comment:
similar comment as in /reservations

https://stackoverflow.com/questions/13366730/proper-rest-response-for-empty-table